src/components/screens/wallet/delegateProfile/performanceView.js

```diff
@@ -18,7 +18,7 @@ export const getStatus = (data) => {
     return data.status;
   }
   if (data.voteWeight >= DEFAULT_STANDBY_THRESHOLD) {
-    return 'standBy';
+    return 'standby';
   }
   return 'inEligible';
```

Comment:
does this also apply to `inEligible`? would be nice to case them similar

Reply:
I can see that in Lisk Service we use `non-eligible`, although according to our last meeting we decided `ineligible` sounds more familiar to native ears. I think we should fix it there. but as for camelCase-lowercase, I can change it to `ineligible`.